packages/network/test/test-network-misc.js

```diff
@@ -167,6 +168,38 @@ test('handled protocol', async t => {
   await when(port.revoke());
 });
 
+test('verify port allocation', async t => {
+  const zone = makeDurableZone(
+    provideBaggage('network-verify-port-allocation'),
+  );
+  const powers = prepareVowTools(zone);
+  const { when } = powers;
+  const makeNetworkProtocol = prepareNetworkProtocol(zone, powers);
+  const makeEchoConnectionHandler = prepareEchoConnectionKit(zone);
+  const makeProtocolHandler = prepareProtocolHandler(
+    zone,
+    t,
+    makeEchoConnectionHandler,
+    powers,
+  );
+  const protocol = makeNetworkProtocol(makeProtocolHandler());
+  const makePortAllocator = preparePortAllocator(zone, powers);
+  const portAllocator = makePortAllocator({ protocol });
+
+  const ibcPort = await when(portAllocator.allocateIBCPort());
+  t.is(ibcPort.getLocalAddress(), '/ibc-port/1');
```

Comment:
This doesn't look like the correct result (the [IBC standard states that port identifiers need to be at least 2 characters in length](https://github.com/cosmos/ibc/blob/main/spec/core/ics-024-host-requirements/README.md#paths-identifiers-separators)).  The local address should be `/ibc-port/port-1`.  Maybe a `port-` prefix got dropped somewhere in the PRs?

Reply:
We actually mock the [generatePortID](https://github.com/Agoric/agoric-sdk/blob/b78d824f3866bd1bd2bc2726e65659d89ed3cfdb/packages/network/test/test-network-misc.js#L54) method, and in that implementation, we drop the `port` prefix. I've updated it to be inline with what we do in production